lbrynet/core/BlobAvailability.py

```diff
@@ -1,4 +1,6 @@
 import logging
+import random
+import time
```

Comment:
These should probably be imported from `lbrynet.core.utils utils for easier testing